components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIProviderImplTest.java

```diff
@@ -2747,17 +2749,18 @@ public void testDeleteAPI_WFException() throws RegistryException, UserStoreExcep
         Mockito.when(wfe.getWorkflowExecutor(WorkflowConstants.WF_TYPE_AM_API_STATE)).
                 thenThrow(WorkflowException.class);
 
-        apiProvider.deleteAPI(identifier);
+        apiProvider.deleteAPI(identifier, apiUuid);
     }
 
     @Test
     public void testDeleteAPI_WithActiveSubscriptions() throws RegistryException, UserStoreException,
             APIManagementException, WorkflowException {
         APIIdentifier identifier = new APIIdentifier("admin-AT-carbon.super", "API1", "1.0.0");
+        String apiUuid = "12345w";
```

Comment:
What's this?

Reply:
It's a sample api_UUID for testing purpose

Reply:
Same as above